src/components/ProjectSettingsTabs/NotificationsTab/types.ts

```diff
@@ -0,0 +1,162 @@
+import {
+  buildRegexSelectors,
+  taskRegexSelectors,
+  requesterSubscriberConfig,
+  ResourceType,
+  Trigger,
+  failureTypeSubscriberConfig,
+} from "constants/triggers";
+import { validateDuration, validatePercentage } from "utils/validators";
+
+export const triggers: Trigger[] = [
```

Comment:
This should be in a constants file since this isn't a typescript type.

Reply:
done